tests/Integration/WhatsAppCloudApiTest.php

```diff
@@ -301,6 +301,25 @@ public function test_send_reply_buttons()
         $this->assertEquals(false, $response->isError());
     }
 
+    public function test_send_single_product()
+    {
+        $catalog_id = '<catalog-id>';
```

Comment:
It is possible to create some object to retrieve a `catalog_id` and a `product_sku_id`?

Reply:
May be to get the catalog_id and product_sku_id from the sample test credentials file? If that's what you mean I can proceed to do that.